packages/mui-joy/src/Button/Button.tsx

```diff
@@ -178,21 +223,28 @@ Button.propTypes /* remove-proptypes */ = {
    * Either a string to use a HTML element or a component.
    * This is equivalent to `components.Root`. If both are provided, the `component` is used.
    */
-  component: PropTypes /* @typescript-to-proptypes-ignore */.elementType,
+  component: PropTypes.elementType,
   /**
    * If `true`, the button will take up the full width of its container.
    * @default false
    */
   fullWidth: PropTypes.bool,
+  /**
+   * The border-radius of the component.
+   * @default 'sm'
+   */
+  radius: PropTypes.oneOf(['lg', 'md', 'sm', 'xl', 'xs']),
```

Comment:
```suggestion
  radius: PropTypes.oneOf(['lg', 'md', 'sm', 'xl', 'xs']),
```
Which of these would a developer use for a square button?

Reply:
`radius` prop is removed.